playbooks/simple/filesystem_modify.yml

```diff
@@ -0,0 +1,16 @@
+---
+- hosts: localhost
+  vars_files:
+    - ../../group_vars/all
+  tasks:
+    - name: Modify filesystem
+      fileInterface:
+        login:
+          unityIP: "{{ unity_host }}"
```

Comment:
Same

Reply:
You are right. By the way, it invlolves your commit as well